ttn-lw-stack.yml

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
?

Reply:
And maybe add a note to keep this in sync with `doc/content/guides/getting-started/configuration.md`